rare_api/views/post.py

```diff
@@ -152,7 +152,6 @@ class PostSerializer(serializers.ModelSerializer):
     rare_user = RareUserSerializer(many=False)
     category = CategorySerializer(many=False)
     tags = TagSerializer(many=True)
-    reactions = ReactionSerializer(many=True)
```

Comment:
Wow what a silly little thing for **someone** to have forgotten! Whoever did that should think about what they've done. Thanks for fixing it.